oauth/index.js

```diff
@@ -23,9 +24,11 @@ acceptField.alg = acceptAlg;
 var productOnly;
 var cacheKey = false;
 
+//const apiKeyCache = "apiKeyCache";
```

Comment:
Is there a reason you want to keep the `apiKeyCache` commented out instead of removing it?

Reply:
done. please see below